deployments/common/image/environments/frozen/ds9-frozen.yml

```diff
@@ -0,0 +1,151 @@
+name: ds9
```

Comment:
The DS-9 kernel is now gone and replaced by the Ubuntu package for DS-9.   This should not be here.